### DISCUSSION TITLE
How to change the static margin on the buttom progress bar?

### DISCUSSION THREAD

**Discussion by zenminimalist** [koreader/koreader#12235] [Q&A] [answered]
![IMG_1454](https://github.com/user-attachments/assets/094b1457-99fc-4c5f-b560-4ef1103a0fc0)

I had this setting set to 10, but someone accidentally changed it. Now, I can't find where to change it back to the original setting I had, which was "10".

I'm basically trying to _reduce_ the left & right space on the borders of the progress bar. Like so:
![IMG_1455](https://github.com/user-attachments/assets/fc2f7337-a66d-419f-84d9-c418186b22da)

**Anyone know how/where to change it?** Is it under advanced settings with the 8 pages of config parameters? If so, which one of these parameters is it?

**Comment by Frenzie** [comment]
That was changed in https://github.com/koreader/koreader/pull/6260 four years ago. Did you update with a very large gap in between?

**Reply by zenminimalist** [reply]
Nope, I updated to every update in between. I had this on my Kindle. On my Supernote, I had the same settings until I accidentally changed it. 

I checked, but don't have a` /frontend/apps/reader/modules/` directory. How do I change without changing the whole book margin?

**Reply by Frenzie** [reply]
It's not a setting that can be changed at this time. On Android it comes from the system, everywhere else it's 10.

**Reply by zenminimalist** [reply]
:( Any plans to allow this? It would make sense to allow users to make it match the Alt status bar that goes edge to edge. That would make it consistent for those who want it. 

Also, adding a `|` between items in the alt bar and allowing you to change the arrangement, would make it consistent with the configuration of the progress bar.

Or any tweak, work-around for it to work (obviously not going to be officially supported that way, I know) 😉

**Reply by Frenzie** [reply, marked as answer]
Not from me, but you could open an issue about it following the issue template. It's been suggested here before: https://github.com/koreader/koreader/issues/6157#issuecomment-629757688
